classes/Pods.php

```diff
@@ -1035,89 +1091,73 @@ public function field( $name, $single = null, $raw = false ) {
 					}
 
 					$object_field_found = true;
-				}
-			}
 
-			if ( ! $object_field_found ) {
-				foreach ( $image_fields as $image_field ) {
-					if ( isset( $this->fields[ $image_field ] ) ) {
-						// Skip any names that are registered fields within this Pod.
-						continue;
-					}
-					if (
-						$image_field === $params->name ||
-						0 === strpos( $params->name, $image_field . '.' )
-					) {
-						// Is it a URL request?
-						$url = '_url' === substr( $image_field, -4 );
-						if ( $url ) {
-							$image_field = substr( $image_field, 0, -4 );
-						}
-
-						// Results in an empty array if no traversal names are passed.
-						$traverse_names = explode( '.', $params->name );
-						array_shift( $traverse_names );
-
-						$attachment_id = 0;
-						switch ( $image_field ) {
-							case 'post_thumbnail':
-								$attachment_id = get_post_thumbnail_id( $this->id() );
-								break;
-							case 'image_attachment':
-								if ( isset( $traverse_names[0] ) ) {
-									$attachment_id = $traverse_names[0];
-									array_shift( $traverse_names );
-								}
-								break;
-						}
+				} elseif ( in_array( $first_field, $image_fields, true ) ) {
+					// Default image field handlers.
 
-						if ( $attachment_id ) {
-
-							$size = 'thumbnail';
-							if ( isset( $traverse_names[0] ) ) {
-								$size  = $traverse_names[0];
-								$sizes = get_intermediate_image_sizes();
-								// Not shown by default.
-								$sizes[] = 'full';
-								$sizes[] = 'original';
-								if ( ! in_array( $size, $sizes, true ) ) {
-									// No valid image size found.
-									$size = false;
-								}
-							}
+					$image_field = $first_field;
+					// Is it a URL request?
+					$url = '_url' === substr( $image_field, - 4 );
+					if ( $url ) {
+						$image_field = substr( $image_field, 0, - 4 );
+					}
 
-							if ( $url ) {
-								$value = pods_image_url( $attachment_id, $size, 0, true );
+					// Copy traversal parameters.
+					$traverse_params = $traverse_fields;
+					// Results in an empty array if no traversal params are passed.
+					array_shift( $traverse_params );
 
-								$object_field_found = true;
-								break;
+					$attachment_id = 0;
+					switch ( $image_field ) {
+						case 'post_thumbnail':
+							$attachment_id = get_post_thumbnail_id( $this->id() );
+							break;
+						case 'image_attachment':
+							if ( isset( $traverse_params[0] ) ) {
+								$attachment_id = $traverse_params[0];
+								array_shift( $traverse_params );
 							}
+							break;
+					}
 
-							if ( $size ) {
-								// Pods will auto-get the thumbnail ID if this isn't an attachment.
-								$value = pods_image( $attachment_id, $size, 0, null, true );
-
-								$object_field_found = true;
-								break;
+					if ( $attachment_id ) {
+
+						$size = 'thumbnail';
+						if ( isset( $traverse_params[0] ) ) {
+							$size  = $traverse_params[0];
+							$sizes = get_intermediate_image_sizes();
```

Comment:
:no_entry_sign: Intermediate images do not exist on the VIP platform, and thus get_intermediate_image_sizes() returns an empty array() on the platform. This behavior is intentional to prevent WordPress from generating multiple thumbnails when images are uploaded.

<sub>hash: 687b78d405c8758f30b022264878143d</sub>